openedx/adg/lms/tests/test_helpers.py

```diff
@@ -23,3 +27,16 @@ def test_get_user_first_name(first_name, full_name, expected_first_name):
     UserProfileFactory(user=user, name=full_name)
 
     assert get_user_first_name(user) == expected_first_name
+
+
+@pytest.mark.django_db
+def test_convert_date_time_zone_and_format():
+    """
+    Test convert_time_zone helper
+    """
+    webinar = WebinarFactory(start_time=datetime(2020, 1, 1, 13, 10, 1))
+
+    assert convert_date_time_zone_and_format(
+        webinar.start_time, WEBINAR_DEFAULT_TIME_ZONE,
+        WEBINAR_DATE_TIME_FORMAT
+    ) == 'Wednesday, January 01, 2020 04:10 PM AST'
```

Comment:
This would be much cleaner:

```
    expected_date_time = 'Wednesday, January 01, 2020 04:10 PM AST'
    actual_date_time = convert_date_time_zone_and_format(
        webinar.start_time, WEBINAR_DEFAULT_TIME_ZONE, WEBINAR_DATE_TIME_FORMAT
    )

    assert expected_date_time == actual_date_time
```